Gemfile

```diff
@@ -6,10 +6,13 @@ gem 'spring', :group => :development
 gem 'pg'
 gem 'figaro'
 gem "faraday"
+gem "twitter"
+gem 'whenever', :require => false
```

Comment:
This is for scheduling cron jobs. I set it up to pull new tweets (the method uses find_or_create_by on the most recent 500 tweets) every 15 mins.

I *think* it should be working, but i need to do some digging before I'm 100% sure.